src/main/java/org/jenkinsci/plugins/jiraext/view/Transition.java

```diff
@@ -50,27 +50,48 @@ public Transition(String transitionName)
 
     @Override
     public void perform(List<JiraCommit> jiraCommitList,
-                        AbstractBuild build, Launcher launcher, BuildListener listener)
-    {
-        try
-        {
-            for (JiraCommit jiraCommit : JiraCommit.filterDuplicateIssues(jiraCommitList))
-            {
+                        AbstractBuild build, Launcher launcher, BuildListener listener) {
```

Comment:
There isn't a style guide but try to be consistent with existing style - IDE might be doing this on your behalf? Brace should be on newline